sw/otbn/crypto/p256_ecdsa_sca.s

```diff
@@ -33,33 +33,60 @@ p256_ecdsa_verify:
   jal      x1, p256_verify
   ecall
 
-.bss
+.data
 
 /* Freely available DMEM space. */
 
 /* Operation mode (1 = sign; 2 = verify) */
 .globl mode
 .balign 4
 mode:
-  .zero 4
+  .word 0x1
 
 /* All constants below must be 256b-aligned. */
 
 /* random scalar k (in two shares) */
 .global k0
 .balign 32
 k0:
-  .zero 32
+  /*little-endian: 0x00000000...ffffffff*/
```

Comment:
I wouldn't use the term little-endian here because both formats are little endian and  the endianness is just about the byte order within a word. If I understand correctly you want to say that the word order is from last word (Line 53) to first word (Line 60), don't you? If yes, I would just say something like:
> /*  Note that the word with the lowest index is last, i.e., k0[0] = 0x0000000, ... k0[7]=0xffffffff */"

Reply:
Thanks for changing this.